.github/workflows/check-drone-signature.yaml

```diff
@@ -0,0 +1,52 @@
+name: Drone CI Signature Check
+
+on:
+  workflow_call:
+    inputs:
+      drone_config_path:
+        description: "Path to the Drone CI configuration file"
+        type: string
+        default: ".drone.yml"
+      drone_server:
+        description: "Drone CI server URL"
+        type: string
+        default: "https://drone.grafana.net"
+
+jobs:
+  check-drone-signature:
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
+
+      - name: Set up Drone CLI
+        run: |
+          curl -L https://github.com/drone/drone-cli/releases/latest/download/drone_linux_amd64.tar.gz | tar zx
```

Comment:
Should this perhaps be pinned to a specific version of the CLI?

Reply:
I'm not sure drone supports downloading a specific version. See [installation instructions docs](https://docs.drone.io/cli/install/).

Reply:
You should be able to do something to infer the version. IE I can download a specific version with a URL like this: https://github.com/harness/drone-cli/releases/download/v1.8.0/drone_linux_amd64.tar.gz

So you should be able to parameterize the version.

Reply:
Awesome, thank you 🙂